mantidproject/mantid
# Reflectometry GUI now outputs stitched transmission run in wavelength

The Reflectometry GUI was supposed to output a stitched transmission in wavelength if transmission runs are specified before processing. This happens if transmission runs were specified directly in each row being processed, but not if they were specified in the `Settings` tab. This PR fixes this very issue.

**To test:**

- Open the Reflectometry interface from Interfaces->Reflectometry->ISIS Reflectometry
- Load `INTER_NR_test2.tbl` (available from sample ISIS dataset) by navigating to Reflectometry->Import .TBL
- Opening the first group reveals they have transmission runs `13463` and `13464`. Processing the group should produce a stitched transmission workspace `TRANS_13463_13464`.
- Remove the transmission runs from the table. Go to the `Settings` tab and input `13463,13464` into `Transmission run(s)`. Go back to the `Runs` tab and process the group again and ensure you have the same results as before.

Fixes #18707.

*Does not need to be in the release notes.*

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
